.github/workflows/macos-xpack-osquerybeat.yml

```diff
@@ -1,4 +1,4 @@
-name: x-pack-heartbeat
+name: x-pack-osquerybeat
```

Comment:
I assume that these and similar changes are just to align the file name and the name in the YAML and that they are not related to the telemetry traces?

Reply:
Exactly, I mixed up two different concerns in this PR, I'll add a note

Reply:
Changes are not related to the telemetry traces but a bug in the folder name